basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
```

Comment:
I asked because you run `eminem regenerate` on this path, so to me it seems it can only be in the `~/.local/share/com.endlessm.subscriptions` area?